src/api.js

```diff
@@ -60,11 +61,11 @@ export default class Api {
    * @return {String}
    */
   endpoints(options={fullUrl: true}) {
-    var root = options.fullUrl ? this.remote : `/${this.version}`;
+    var root = options.fullUrl ? `${this.remote}/` : `/${this.version}/`;
     var urls = {
       root:                   () => root,
```

Comment:
I would have put the trailing slash here, so that assembling urls remains consistent (with ``/`` as join char)